src/commands/subscription.ts

```diff
@@ -18,6 +18,17 @@ export const emitter = new ChessLeagueEmitter()
 type Context = any
```

Comment:
We should get rid of these. :(  Fully meant to do that back when I first wrong them.  we can turn on eslint no-explicit-any and fix from there.

Reply:
In this case, I think you should probably go ahead and get rid of this any for this change, to make sure we're probably using the context throughout.